### DISCUSSION TITLE
Creating a New User

### DISCUSSION THREAD

**Discussion by mathworldtesting** [ludeknovy/jtl-reporter#250] [General]
I am trying to create a new user and I keep getting Unexpected error occurred: f31e1ae1-d6e3-4751-9737-6661e27bf2fc. I am using the typical pattern for my password [a-zA-Z0-9]. With either no special characters or either ! or @.

![image](https://github.com/ludeknovy/jtl-reporter/assets/142104984/812ea7f9-1e97-4283-b732-f1453c4b448d)

**Comment by ludeknovy** [comment]
Hi @mathworldtesting. Most likely the db was not properly initialized. Please try to start the app again and wait a bit. If it won’t help, please send me over the longs from the **be** container

**Reply by ludeknovy** [reply]
@mathworldtesting did you resolve the issue?
